### PR TITLE
[Merged by Bors] - Add quad to 2d_shapes example

### PR DESCRIPTION
**Pull request by lewiszlw**
# Objective

- Add quad to 2d_shapes example

### CODE REVIEW

File: examples/2d/2d_shapes.rs [hunk 1]
```diff
@@ -34,11 +34,21 @@ fn setup(
...
+        transform: Transform::from_translation(Vec3::new(100., 0., 0.)),
+        ..default()
+    });
+
     // Hexagon
     commands.spawn(MaterialMesh2dBundle {
         mesh: meshes.add(shape::RegularPolygon::new(50., 6).into()).into(),
         material: materials.add(ColorMaterial::from(Color::TURQUOISE)),
-        transform: Transform::from_translation(Vec3::new(100., 0., 0.)),
+        transform: Transform::from_translation(Vec3::new(200., 0., 0.)),
```
Comment (rparrett) [inline examples/2d/2d_shapes.rs:51]: Can we make sure that the entire collection of shapes remains evenly spaced and centered on the screen?

File: examples/2d/2d_shapes.rs [hunk 2]
```diff
@@ -34,11 +34,21 @@ fn setup(
         ..default()
     });
 
+    // Quad
+    commands.spawn(MaterialMesh2dBundle {
+        mesh: meshes
+            .add(shape::Quad::new(Vec2::new(50., 100.)).into())
+            .into(),
+        material: materials.add(ColorMaterial::from(Color::BLUE)),
```
Comment (rparrett) [inline examples/2d/2d_shapes.rs:42]: Full-blue is pretty jarring next to the other colors which are a bit muted. Maybe something like `LIME_GREEN`?

**Review by rparrett — APPROVED** [review]
This seems like a good idea. Users looking at the previous version of this example might be lead to believe that sprites are the one and only way to draw rectangles, or that they need to use a 4-sided `RegularPolygon`.

**Review by james7132 — APPROVED** [review]

**Review by mockersf — APPROVED** [review]

**Pull request closed without merging**